qemu/tests/cfg/drive_mirror.cfg

```diff
@@ -34,14 +34,14 @@
     tmp_dir = "/tmp"
     repeat_times = 3
     cancel_timeout = 5
+    #For drive_mirror_complete, target image will not be boot by default, as boot_target_image=no
```

Comment:
@xutian thanks, but not only drive_mirror need this, I prefer add them together for all block jobs later, handle this separately.